common/server.h

```diff
@@ -187,8 +187,9 @@ private slots:
     void sendIsl_GameCommand(const CommandContainer &item, int serverId, qint64 sessionId, int roomId, int playerId);
     void sendIsl_RoomCommand(const CommandContainer &item, int serverId, qint64 sessionId, int roomId);
 
-    void addExternalUser(const ServerInfo_User &userInfo);
-    void removeExternalUser(const QString &userName);
+    virtual void addExternalUser(const ServerInfo_User &userInfo) = delete;
+    virtual void removeExternalUser(const QString &userName) = delete;
```

Comment:
These looks like leftovers as they have been implemented in `Server_Room`, maybe they can just be deleted.

Reply:
Gotcha